setup.cfg

```diff
@@ -30,3 +30,12 @@ console_scripts =
 
 [wheel]
 universal = 1
+
+[flake8]
+builtins = _
+show-source = True
+exclude=.venv,.git,.tox,dist,doc,*openstack/common*,*lib/python*,*egg,build
+# Minimal config needed to make flake8 compatible with black output:
+max-line-length=160
+# See https://github.com/PyCQA/pycodestyle/issues/373
+ignore = E203, W503
```

Comment:
I would use `extend-ignore = E203` instead personally

Reply:
Agreed. This is the better way to do it.